drivers/gpio/gpio_nrfx.c

```diff
@@ -493,27 +493,37 @@ static void gpiote_event_handler(void)
 	}
 }
 
+#define GPIOTE_NODE DT_INST(0, nordic_nrf_gpiote)
+
 static int gpio_nrfx_init(struct device *port)
 {
 	static bool gpio_initialized;
 
 	if (!gpio_initialized) {
 		gpio_initialized = true;
-		IRQ_CONNECT(DT_NORDIC_NRF_GPIOTE_GPIOTE_0_IRQ_0,
-			    DT_NORDIC_NRF_GPIOTE_GPIOTE_0_IRQ_0_PRIORITY,
+		IRQ_CONNECT(DT_IRQN(GPIOTE_NODE), DT_IRQ(GPIOTE_NODE, priority),
 			    gpiote_event_handler, NULL, 0);
 
-		irq_enable(DT_NORDIC_NRF_GPIOTE_GPIOTE_0_IRQ_0);
+		irq_enable(DT_IRQN(GPIOTE_NODE));
 		nrf_gpiote_int_enable(NRF_GPIOTE, NRF_GPIOTE_INT_PORT_MASK);
 	}
 
 	return 0;
 }
 
+/*
+ * Device instantiation is done with node labels because 'port_num' is
```

Comment:
I wonder about just having a 'port' property in dts for this case?  Since this info is clearly needed by the CONFIG register in gpiote

Reply:
The same argument could be made for all of our bindings, and apparently some of Atmel's too according to what I understood from Gerson on Slack.

Given that this is a single instance of a wider issue, I don't want to do ad hoc work here that's out of the scope of just moving to the new API.